src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -173,7 +176,9 @@ bool convert_precision(ov::pass::PassBase& pass,
             // If output type mismatch given type we try to fuse type into this operation
             // otherwise we insert Convert operation.
             for (auto& node : ops) {
-                pass.transformation_callback(node);
+                if (skip_precision_sensitive && fp16_compression_is_disabled(node))
+                    continue;
```

Comment:
why can't we use just condition `fp16_compression_is_disabled(node)`?

Reply:
if we do so when plugins will call `ConvertPrecision` directly to align types to 32 or 8 bit element types, `i64, u64, i16, u16` and other types happened to be marked as `fp16_compression_is_disabled` will be left in in `i64, u64...` and so on, this can result in type inconsistency error

https://github.com/openvinotoolkit/openvino/blob/e306cbc67afb54fba7620dc6669e5cb9ea2d60f8/src/plugins/intel_cpu/src/transformation_pipeline.cpp#L189-L236

Reply:
Let us discuss this logic tomorrow. Now it looks a bit "kostylno":)
I thought that marking passes will mark nodes which is sensitive and must not be affected by the compression. And for that nodes `fp16_compression_is_disabled` returns true and it will be sufficient.
Now a bit overcomplication remains:)
Will discuss this tomorrow.

Reply:
Discussed personally. Decided to leave this `skip_precision_sensitive` flag